lib/ExpensiMark.ts

```diff
@@ -658,13 +697,38 @@ export default class ExpensiMark {
 
             {
                 name: 'image',
-                regex: /<img[^><]*src\s*=\s*(['"])(.*?)\1(?:[^><]*alt\s*=\s*(['"])(.*?)\3)?[^><]*>*(?![^<][\s\S]*?(<\/pre>|<\/code>))/gi,
-                replacement: (_extras, _match, _g1, g2, _g3, g4) => {
-                    if (g4) {
-                        return `![${g4}](${g2})`;
+                regex: /<img[^><]*src\s*=\s*(['"])(.*?)\1(.*?)>(?![^<][\s\S]*?(<\/pre>|<\/code>))/gi,
```

Comment:
Can you please "breakdown" the regex changes you made here? I see you have taken out the altRegex, but I can also see some other changes in latter part of the regex

Reply:
- removed`(?:[^><]*alt\s*=\s*(['"])(.*?)\3)?` in favor of `(.*?)` to capture all attributes after src (including alt)
- simplified closing tag matching (`[^><]*>*` to `(.*?)>`

and that's pretty much it.